third_party/gpus/cuda_configure.bzl

```diff
@@ -120,13 +120,33 @@ def _enable_cuda(repository_ctx):
   return False
 
 
-def _cuda_toolkit_path(repository_ctx):
-  """Finds the cuda toolkit directory."""
+def _cuda_toolkit_path(repository_ctx, cuda_version):
+  """Finds the cuda toolkit directory.
+
+   If cuda_version is given, attempts to return a real path.
```

Comment:
Seems slightly redundant since it still attempts to find a real path even if cuda_version is not given.